File: Programmers/JS/p12928.js
/**
 * 약수의 합
 */

function solution(n) {
  let sum = 0;
  for (let i = 1; i <= n; i++) {
    if (n % i === 0) sum += i;
  }
  return sum;
}

const input = 12;

console.log(solution(input));

File: Programmers/C++/p12928.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(int n) {
  int sum = 0;

  for (int i = 1; i <= n; i++){
    if (n % i == 0) {
      sum += i; 
    }
  }
  return sum;
}

int main() {
  int n, result;
  cin >> n;

  cout <<  solution(n) << "\n";

  return 0;
}